docs/topics/mta-web-administrator-view.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-web-administration-view_{context}"]
+= User Interface views
+
+The {ProductName} ({ProductShortName}) {WebName} and has two user interfaces with different functionalities:
+
+* Administrator view
+* Developer view
+
+== Administrator view
+The administrator view is used by administrators of {ProductShortName} {WebName} to manage the configuration of an instance.
+//![](/Tackle2/Views/AdminMenu.png)
+
+=== Credentials
+The credentials management module enables administrators to create and manage credentials to access private repositories.
+//![](/Tackle2/Views/CredMain.png)
+
+The credentials page displays the available credentials, *Edit* and *Delete* buttons and the following fields:
+
+* *Name*
+* *Description*
+* *Type*
+** *Source Control*
+** *Username/Password*
+** *Source Private Key/Passphrase*
+** *Maven Settings File*
+** *Proxy*
+* *Type specific information*
+* *Created by*
+
```

Comment:
See the example at the end of https://redhat-documentation.github.io/supplementary-style-guide/#admonitions to see how to format notes. If you haven't read the entire article, do so in the near future.
https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Reply:
Thank you. My bad.